packages/nativescript-accessibility-ext/utils/utils.android.ts

```diff
@@ -127,4 +127,4 @@ export class AccessibilityServiceEnabledObservable extends CommonA11YServiceEnab
   }
 }
 
-export * from '@nativescript/core/utils/utils';
+export * from '@nativescript/core/utils';
```

Comment:
**Facing error:**

ERROR in ./node_modules/@nota/nativescript-accessibility-ext/utils/utils.android.js 96:0-47
Module not found: Error: Can't resolve '@nativescript/core/utils/utils' in '../node_modules/@nota/nativescript-accessibility-ext/utils'
resolve '@nativescript/core/utils/utils' in '../node_modules/@nota/nativescript-accessibility-ext/utils'
  Parsed request is a module
  using description file: ../node_modules/@nota/nativescript-accessibility-ext/package.json (relative path: ./utils)
    resolve as module